Makefile

```diff
@@ -23,6 +27,9 @@ VALE_CMD  ?= $(DOCKER_CMD) $(DOCKER_ARGS) --volume "$${PWD}"/docs/modules:/pages
 VALE_ARGS ?= --minAlertLevel=error --config=/pages/ROOT/pages/.vale.ini /pages
 
 
+COMMODORE_CMD  ?= $(DOCKER_CMD) $(DOCKER_ARGS) projectsyn/commodore:latest component compile .
```

Comment:
Can we change `projectsyn/commodore:latest` to `docker.io/projectsyn/commodore:latest` because we can not assume, that docker.io will be the default registry all the times.

Reply:
Should Docker ever change the default registry in the docker client, this will become a HUGE fallout in thousands of CI/CD pipelines etc, one that we definitely get via news/blogs first. But yes, we can make it explicit, there's no reason not to.